src/lib/Behat/PageObject/PageObjectFactory.php

```diff
@@ -28,6 +29,12 @@ public static function createPage(UtilityContext $context, string $pageName): Pa
                 return new DashboardPage($context);
             case ContentStructurePage::PAGE_NAME:
                 return new ContentStructurePage($context);
+            case ContentTypeGroupsPage::PAGE_NAME:
+                return new ContentTypeGroupsPage($context);
+            case UpdateItemPage::PAGE_NAME:
+                return new UpdateItemPage($context);
+            case ContentTypeGroupPage::PAGE_NAME:
+                return new ContentTypeGroupPage($context, $parameters[0]);
```

Comment:
And I see now how the `ElementFactory` appeared, since here we have already existing similar code.
Still not a huge fan, but let's discuss it internally maybe.